### DISCUSSION TITLE
How to override built-in plugins?

### DISCUSSION THREAD

**Discussion by latipun7** [sxyazi/yazi#1513] [1. Q&A] [answered]
Hello👋

I'd like to override built-in plugins entirely, for example [`file.lua`](https://github.com/sxyazi/yazi/blob/main/yazi-plugin/preset/plugins/file.lua).

Previously, I was able to do this with creating `~/.config/yazi/plugins/file.yazi/init.lua` file, so I don't need to set config in `yazi.toml` to use custom plugin. But I don't remember since when, that drop-in replacement plugin didn't work now.

How to override built-in plugins now?

**Comment by AnirudhG07** [comment]
If you are building from source, then you can try removing the file from the git cloned repo and build. That should work(didnt try). Dont forget to add your file.yazi, else it might throw some error while running yazi

**Comment by sxyazi** [comment, marked as answer]
Hi, that's a breaking change in v0.3, https://github.com/sxyazi/yazi/issues/1046#issuecomment-2140320472, you need to use a different name from the builtin one such as `my-file`, and add 

```toml
[[plugin.append_previewers]]
name = "*"
run  = "my-file"
```

to your `yazi.toml`

**Reply by latipun7** [reply]
Thank you. Sorry, I miss that info about the names of built-in plugins are now reserved 🙏
